proto/result.go

```diff
@@ -0,0 +1,434 @@
+// Copyright 2022 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+package proto
+
+import (
+	"fmt"
+
+	timestamppb "google.golang.org/protobuf/types/known/timestamppb"
+)
+
+// Claim values as defined by
+// https://datatracker.ietf.org/doc/draft-ietf-rats-ar4si/
+const (
+	// common
```

Comment:
What does //common here means?

Reply:
The group of consts bellow are common to all claims. (The subsequent groups are titled with the claim they're for).